app/models/direct_award.py

```diff
@@ -81,12 +100,15 @@ class DirectAwardSearch(db.Model):
     archived_services = db.relationship(ArchivedService, secondary='direct_award_search_result_entries',
                                         order_by=lambda: DirectAwardSearchResultEntry.id, lazy='dynamic')
 
-    project = db.relationship(DirectAwardProject)
+    project = db.relationship(DirectAwardProject, backref="searches")
     user = db.relationship(User)
 
-    # Partial index on project_id,active==1. Enforces only one active search per project at a time.
-    __table_args__ = (db.Index('idx_project_id_active', project_id, active, unique=True,
-                               postgresql_where=db.Column('active')),)
+    __table_args__ = (
+        # this may appear tautological, but is required to be able to make a compound foreign key to these together
```

Comment:
- this may appear tautological, but is required to be able to make a compound foreign key to these together
+ this is required to be able to make a compound foreign key

Or something.

Reply:
I might reword it but want to keep in the information that the casual reader might see it as pointless and want to delete it.

- [ ] TODO

Reply:
I don't understand what we're compounding here, if there's actually only one column. Definitely agree this thing needs some more explanation, appreciate that underneath it all is some sqlalchemy limitation.

Reply:
There's two columns - `id` and `project_id`.